blocks/v2-custom-form/v2-custom-form.js

```diff
@@ -557,6 +557,22 @@ function decorateTitles(block) {
   }
 }
 
+function createHoneypotField() {
+  const fragment = document.createRange().createContextualFragment(`
+    <div class="visually-hidden" aria-hidden="true">
+      <label for="pardot_extra_field">Comments</label>
```

Comment:
Can you update "pardot" to "form", because even if we are using MCAE, form is generic enough to be used in another form handler